[Term Entry] PHP Math Functions pow() (#2122)
Codecademy/docs: content/php/concepts/math-functions /terms/pow/pow.md

---
Title: 'pow()'
Description: 'Returns the number raised to the power of the exponent.'
Subjects: 
  - 'Web Development
  - 'Code Foundations'
Tags: 
  - 'Methods'
  - 'Arithmetic'
  - 'Numbers'
  - 'Functions'
CatalogContent:
  -'learn-PHP'
  -' paths/web-development
--- 

The **`pow()`** function returns the number raised to the power of the exponent.

## Syntax 

```pseudo 
pow ( number $base, number $exp ) : number
``` 

> The `pow()` function takes up to two parameters: 

Reviewer: ```suggestion
The `pow()` function takes two parameters: 
```

-`base` is the number to be raised.
-`exp` is the power for a base to be raised. 

> 

Reviewer: In the syntax section, show the syntax and describe the use of the `**` operator since you mention it in the example.

## Example 

The following example calculates 5^2:

```php
<?php
echo "pow(5,2) = " . pow(5,2);

> echo " using ** operator ". 5**2;

Reviewer: ```suggestion
echo ", using ** operator =" . 5**2;
```

?> 
``` 

The example will result in the following output:

```shell 

> pow(5,2) = 25 using ** operator 25

Reviewer: ```suggestion
pow(5,2) = 25, using ** operator = 25
```

``` 

## Codebyte Example

> The following example calculates any number to the power of 0:

Reviewer: ```suggestion
The following example calculates 5 to the power of 0:
```

---
Title: 'pow()'
Description: 'Returns the number raised to the power of the exponent.'
Subjects: 
  - 'Web Development'
  - 'Code Foundations'
Tags: 
  - 'Methods'
  - 'Arithmetic'
  - 'Numbers'
  - 'Functions'
CatalogContent:
  -'learn-PHP'
  -'paths/web-development'
--- 

The **`pow()`** function returns the number raised to the power of the exponent.

## Syntax 

```pseudo 
pow ( number $base, number $exp ) : number 
``` 

The `pow()` function takes two parameters: 

-`base` is the number to be raised.
-`exp` is the power for a base to be raised.
- You can also use the exponent operator `**`.

> - pow($base, $exp) returns base ^ exp, which is the same as base ** exp.  

Reviewer: ```suggestion
- `pow($base, $exp)` returns base ^ exp, which is the same as `base ** exp`.  
```
